aztec/src/main/kotlin/org/wordpress/aztec/spans/HighlightSpan.kt

```diff
@@ -0,0 +1,20 @@
+package org.wordpress.aztec.spans
+
+import android.content.Context
+import android.text.style.BackgroundColorSpan
+import androidx.core.content.ContextCompat
+import org.wordpress.aztec.AztecAttributes
+import org.wordpress.aztec.R
+import org.wordpress.aztec.formatting.InlineFormatter
+
+class HighlightSpan(
+        override var attributes: AztecAttributes = AztecAttributes(),
+        highlightStyle: InlineFormatter.HighlightStyle = InlineFormatter.HighlightStyle(R.color.grey_lighten_10),
+        context: Context) : BackgroundColorSpan(ContextCompat.getColor(context, highlightStyle.color)), IAztecInlineSpan {
```

Comment:
(⛏️ ) Put this in a new line:
```suggestion
        context: Context
) : BackgroundColorSpan(ContextCompat.getColor(context, highlightStyle.color)), IAztecInlineSpan {
```